### ISSUE TITLE
top1 error = 0.99 when testing

### ISSUE THREAD

**Issue by YoungDav** [JiahuiYu/slimmable_networks#48]
python==3.5.6
torch==1.1.0
torchvision==0.4.1

1. mkdir logs/ && Download pretrain-model 
2. ln -s imagenet_datadir  ./
3. `python train.py app:apps/s_mobilenet_v1_train_val.yml` or `python train.py app:apps/us_mobilenet_v1_train_val.yml`
4. Error log:

77.3s   test    1.0     0/480: loss: 20.153, top5_error: 0.997, top1_error: 0.999
77.3s   test    0.75    0/480: loss: 19.319, top5_error: 0.997, top1_error: 0.999
77.4s   test    0.5     0/480: loss: 17.999, top5_error: 0.997, top1_error: 0.999
77.4s   test    0.25    0/480: loss: 15.237, top5_error: 0.996, top1_error: 0.999
66.0s   test    1.0     0/480: loss: 20.153, top5_error: 0.997, top1_error: 0.999
66.0s   test    0.75    0/480: loss: 19.319, top5_error: 0.997, top1_error: 0.999
66.1s   test    0.5     0/480: loss: 17.999, top5_error: 0.997, top1_error: 0.999
66.1s   test    0.25    0/480: loss: 15.237, top5_error: 0.996, top1_error: 0.999

thanks

**Comment by YoungDav** [comment]
error come from **dataset format no match**
